node/sc/remote_backend.go

```diff
@@ -66,29 +65,11 @@ func NewRemoteBackend(sb *SubBridge, rawUrl string) (*RemoteBackend, error) {
 }
 
 func (rb *RemoteBackend) checkConnection() bool {
-	var connected = true
-
-	if rb.subBridge.bridgeServer == nil {
-		return false
-	}
-
-	peers := rb.subBridge.bridgeServer.PeersInfo()
-	if peers == nil || len(peers) < 1 {
+	// It is difficult to determine the exact number of peers, and if there is no peer, timeout will occur.
+	if rb.subBridge.peers.Len() == 0 {
```

Comment:
Instead of peer count checking, can we add a status variable if ResetAllSubscribedEvents is completed?

Reply:
I think that additional flags may make the code unnecessarily complex. checking peer.Len() could be enough here.